src/main/java/org/prebid/server/handler/openrtb2/AmpHandler.java

```diff
@@ -287,7 +289,17 @@ private void handleResult(AsyncResult<AmpResponse> responseResult, AmpEvent.AmpE
             body = Json.encode(responseResult.result());
         } else {
             final Throwable exception = responseResult.cause();
-            if (exception instanceof InvalidRequestException) {
+            if (exception instanceof BlacklistedAppException || exception instanceof BlacklistedAccountException) {
+                metricRequestStatus = exception instanceof BlacklistedAccountException
+                        ? MetricName.blacklisted_account : MetricName.blacklisted_app;
+                final String errorMessage = exception.getMessage();
+                logger.info("Blacklisted: {0}", errorMessage);
```

Comment:
Should it really be logged with `info` level? It threatens that log file to be overwhelmed by such entries if some app or publisher starts shelling PBS with requests.

Reply:
Will then change it to `debug` as I've discussed with @rpanchyk